codebase/superdarn/src.idl/lib/main.1.25/fit.pro

```diff
@@ -255,11 +261,24 @@ function FitRead,unit,prm,fit
       fit.phi0[slist]= (*(arrvec[arrid[30]].ptr))[*]
       fit.phi0_e[slist]= (*(arrvec[arrid[31]].ptr))[*]
       fit.elv[slist]= (*(arrvec[arrid[32]].ptr))[*]
-      fit.elv_low[slist]= (*(arrvec[arrid[33]].ptr))[*]
-      fit.elv_high[slist]= (*(arrvec[arrid[34]].ptr))[*]
-      fit.x_sd_l[slist]= (*(arrvec[arrid[35]].ptr))[*]
-      fit.x_sd_s[slist]= (*(arrvec[arrid[36]].ptr))[*]
-      fit.x_sd_phi[slist]= (*(arrvec[arrid[37]].ptr))[*]
+      fit.elv_low[slist]= (*(arrvec[arrid[35]].ptr))[*]
+      fit.elv_high[slist]= (*(arrvec[arrid[36]].ptr))[*]
+      fit.x_sd_l[slist]= (*(arrvec[arrid[37]].ptr))[*]
+      fit.x_sd_s[slist]= (*(arrvec[arrid[38]].ptr))[*]
+      fit.x_sd_phi[slist]= (*(arrvec[arrid[39]].ptr))[*]
+    endif
+    
+    ; XCF fitted parameters for FitACF 3
+    if (fit.revision.major ge 3) then begin
```

Comment:
@ecbland this value was `5` from ~2004-2019 (and `4` from ~2000-2004), so there may be issues with files generated using previous versions of FITACF / RST / ROS: 

6dd7dd348dde3ad12174a313147f1c4daa0bbb4a
https://github.com/SuperDARN/rst-archive/blob/rst.2.05/code/src.lib/superdarn/fitacf.1.22/src/fitacfversion.h
https://github.com/SuperDARN/ros-archive/blob/ros.0.93A/qnx/code/src.bin/superdarn/process/make_fit.1.04/make_fit.c#L141

Reply:
@egthomas Thanks for reminding me about this :scream: 

I suppose the simplest solution is to change those lines to `if (fit.revision.major eq 3)`, and then if someone ever writes a FitACF 4.0 they can find a different solution. I can leave some comments in the code as well. Does that sound sensible?